Cargo.toml

```diff
@@ -11,21 +11,21 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = {version="0.13", features = ["multi-threaded", "bevy_render", "bevy_core_pipeline", "bevy_asset", "bevy_sprite", "bevy_winit", "png", "x11", "bevy_ui", "tga", "bmp", "jpeg", "webgl2"], default-features=false }
-bevy_egui = "0.25"
-bevy_pancam = { version = "0.11.0", features = ["bevy_egui"] }
-bevy_prototype_lyon = "0.11.0"
+bevy = {version="0.14.0", features = [ "bevy_render", "bevy_core_pipeline", "bevy_asset", "bevy_sprite", "bevy_winit", "png", "x11", "bevy_ui", "tga", "bmp", "jpeg", "webgl2"], default-features=false }
+bevy_egui = "0.29.0"
```

Comment:
We may want to keep it at `0.28`. It is the version that `bevy_pancam`  is using and it would get duplicated.

Reply:
done